specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -994,6 +1034,48 @@
         "tableName"
       ]
     },
+    "RestServiceDataset": {
+      "x-ms-discriminator-value": "RestResource",
+      "description": "A Rest service dataset.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Dataset"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Properties specific to this dataset type.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RestServiceDatasetTypeProperties"
+        }
+      }
+    },
+    "RestServiceDatasetTypeProperties": {
```

Comment:
same here, why isn't this named RestResourceDatasetTypeProperties?

Reply:
@dazhao-msft

Reply:
changed